src/js/components/Diagram/stories/Progressing.js

```diff
@@ -72,13 +72,20 @@ export const Progressing = () => {
               ))}
             </Box>
           </Box>
-          <Diagram connections={connections} />
+          <Diagram
+            animation={{ type: 'draw', duration: 2000 }}
```

Comment:
Will this support things like delay and size?

Are there values are/are not acceptable by these? For example and delay of 3000 ms?

Reply:
Specified where delay works as well as added the size and a comment to show that size currently is only applicable to the pulse animation

Reply:
This information could be helpful for the user and should be added to the docs.